include/deal.II/lac/precondition.h

```diff
@@ -871,13 +871,26 @@ class PreconditionPSOR : public PreconditionRelaxation<MatrixType>
  * matrices. This preconditioner is based on an iteration of an inner
  * preconditioner of type @p PreconditionerType with coefficients that are
  * adapted to optimally cover an eigenvalue range between the largest
- * eigenvalue down to a given lower eigenvalue specified by the optional
- * parameter @p smoothing_range. The typical use case for the preconditioner
- * is a Jacobi preconditioner specified through DiagonalMatrix, which is also
- * the default value for the preconditioner. Note that if the degree variable
- * is set to zero, the Chebyshev iteration corresponds to a Jacobi
- * preconditioner (or the underlying preconditioner type) with relaxation
- * parameter according to the specified smoothing range.
+ * eigenvalue $\lambda_{\max{}}$ down to a given lower eigenvalue
+ * $\lambda_{\min{}}$ specified by the optional parameter
+ * @p smoothing_range. The algorithm is based on the following three-term
+ * recurrence:
```

Comment:
one can also borrow a picture from https://www.dealii.org/developer/doxygen/deal.II/namespaceUtilities_1_1LinearAlgebra.html#a19ce01cf01ee57e2b1237944154646e6 or link to that function from here.